python/vowpalwabbit/sklearn_vw.py

```diff
@@ -7,9 +7,11 @@
 import io
 
 from scipy.sparse import csr_matrix
+import sklearn
+from sklearn.exceptions import NotFittedError
 from sklearn.base import BaseEstimator, RegressorMixin
 from sklearn.linear_model.base import LinearClassifierMixin, SparseCoefMixin
```

Comment:
These two imports will no longer be available in `sklearn >=0.24`. I'm not sure what we'll need to do to replace them

Reply:
Where's the reference for that?

Reply:
Its in the deprecation warning when importing sklearn.linear_model.base using sklearn 0.22. LinearClassifierMixin and SparseCoefMixin aren't in their public API so we'll lose them when they close off their private API

for the public API:
https://scikit-learn.org/dev/modules/classes.html#module-sklearn.linear_model